presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -1058,6 +1061,74 @@ public void testGetTableSchemaException()
     }
 
     @Test
+    public void testGetTableStatsBucketedStringInt()
```

Comment:
It would be better to have those tests as separate PR as they are not really related to hidden-column stuff.
I extracted first commit here https://github.com/Teradata/presto/commit/7fda962ce57f1ff842f202eb0f1bb6a3d13143ab.
And I will merge it as Travis passes.